public/components/common/form/types.ts

```diff
@@ -54,7 +55,7 @@ interface EnhancedField {
   initialValue: any;
   value: any;
   changed: boolean;
-  error: string | null;
+  error: string;
```

Comment:
**thought**: I think this can be:

```ts
error: string | null | undefined;
```

Reply:
Resolved in commit [444ee42](https://github.com/wazuh/wazuh-kibana-app/pull/5554/commits/444ee425622acb0d5365a9efe700ce59e6f28f9a)